contracts/governance/Governor.sol

```diff
@@ -135,7 +135,6 @@ abstract contract Governor is Context, ERC165, EIP712, Nonces, IGovernor, IERC72
      * @dev See {IGovernor-state}.
      */
     function state(uint256 proposalId) public view virtual returns (ProposalState) {
-        // ProposalCore is just one slot. We can load it from storage to stack with a single sload
```

Comment:
This is still true. Are we just considering it not worth to keep the comment?

Reply:
ProposalCore is two slots now, but the rest of the comment is sort of true and relevant so I'll add it back.